src/pages/[platform]/build-a-backend/storage/storagepath/index.mdx

```diff
@@ -0,0 +1,125 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
```

Comment:
this page is added but it's not in the directory.mjs, so it's not going to show up in the nav